Forms/MainForm.cs

```diff
@@ -27,6 +28,10 @@ public partial class MainForm : Form
         public ItemEditForm ItemEditor { get; private set; }
         public StartingItemEditForm StartingItemEditor { get; private set; }
 
+        public readonly string ROOT = Path.GetDirectoryName(System.Reflection.Assembly.GetEntryAssembly().Location) + "\\";
+        public readonly string PresetExt = ".mmrsp";
+        public bool PresetInit = true;
```

Comment:
It's not really clear what this variable does. It seems unnecessary though; you could instead pass a parameter to the ReadPreset function.

Reply:
It's used in the ReadPreset() to know that it is reading the initial preset. It's there to handle the edge case that the preset file used at launch is ever corrupted.